server/lib/socket/socketCtrl.js

```diff
@@ -16,20 +17,34 @@ socketCtrl = {}
             socket.emit('MESSAGE', {type:'NEW_MESSAGE', text: message})
           })
         })
-        socketCtrl.startServerModeWatch(socket)
+        socketCtrl.startDbWatch(socket)
         fufill()
-      } catch (e) {
-        reject(e)
+      } catch (err) {
+        reject(err)
       }
     })
   }
 
-  socketCtrl.startServerModeWatch = (socket) => {
+  socketCtrl.startDbWatch = (socket) => {
     setInterval(() => {
       serverModeCtrl.checkMode()
       .then((mode) => {
         socket.emit('SERVER_MODE', mode[0].server_mode)
       })
+      .then(() => {
+        return serverModeCtrl.checkMessage()
+      })
+      .then((serverMessageData) => {
```

Comment:
Wondering if we should only emit if the data has changed? rather than emitting every second? Or is that what this does already?